### PR TITLE
CONSOLE-4236:  Migrate Dev > Observe > Dashboard Page 

### PR DESCRIPTION
**Pull request by zhuje**
## Overview 
The goal of this PR is to consolidate code for the Developer perspective > Observe > Dashboard page. We will remove the code that renders this page from openshift/console. The page will now be rendered by openshift/monitoring-plugin through this PR: https://github.com/openshift/monitoring-plugin/pull/167. 

## Related CONSOLE Jira Issue: 
https://issues.redhat.com/browse/CONSOLE-4236

## Related Observability UI Jira Issue: 
https://issues.redhat.com/browse/OU-259

## Testing
Must be tested with PR: https://github.com/openshift/monitoring-plugin/pull/167

This PR openshift/console#14192 removes the Developer perspective > Observe > Dashboard page
This PR openshift/monitoring-plugin#167 adds the Developer perspective > Observe > Dashboard page 

**Expected Results:** 
All behaviors should be the same as before the migration. 

image

**Figure 1**. Shows the Developer Perspective > Observe > Dashboard Page

### CODE REVIEW

**Comment by spadgett** [comment]
/hold

The monitoring-plugin incorrectly references translations from console `public` i18n namespace. This needs to be fixed downstream in the plugin. For example:

https://github.com/openshift/monitoring-plugin/blob/main/web/console-extensions.json#L61

**Comment by zhuje** [comment]
Fixing monitoring-plugin which incorrectly references translations from console public i18n namespace in this PR: https://github.com/openshift/monitoring-plugin/pull/173

**Comment by zhuje** [comment]
/unhold 

> The monitoring-plugin incorrectly references translations from console public i18n namespace. This needs to be fixed downstream in the plugin, which should maintain its own translations. For example:
https://github.com/openshift/monitoring-plugin/blob/main/web/console-extensions.json#L61 [1]

[1] https://github.com/openshift/console/pull/14192#issuecomment-2317450254

I am labelling `unhold`because this fix has been merged in the `monitoring-plugin`: https://github.com/openshift/monitoring-plugin/pull/173. This is in response the `/hold` comment above.

**Comment by zhuje** [comment]
/assign 
@spadgett if you have time, could you review this PR?

**Comment by zhuje** [comment]
/assign @spadgett

Can you please review this pull request?

File: frontend/public/reducers/observe.ts [hunk 1]
```diff
@@ -3,12 +3,12 @@ import { List as ImmutableList, Map as ImmutableMap } from 'immutable';
...
 import { ActionType, ObserveAction } from '../actions/observe';
-import {
-  MONITORING_DASHBOARDS_DEFAULT_TIMESPAN,
-  MONITORING_DASHBOARDS_VARIABLE_ALL_OPTION_KEY,
-} from '../components/monitoring/dashboards/types';
 import { isSilenced } from '../components/monitoring/utils';
 
+export const MONITORING_DASHBOARDS_DEFAULT_TIMESPAN = 30 * 60 * 1000;
+
+export const MONITORING_DASHBOARDS_VARIABLE_ALL_OPTION_KEY = 'ALL_OPTION_KEY';
```
Comment (kyoto) [inline frontend/public/reducers/observe.ts:10]:
```suggestion
const MONITORING_DASHBOARDS_DEFAULT_TIMESPAN = 30 * 60 * 1000;

const MONITORING_DASHBOARDS_VARIABLE_ALL_OPTION_KEY = 'ALL_OPTION_KEY';
```

It looks like these constants are now only used in this file.

Moreover, I think the various `ActionType.DashboardsPatchVariable*` Redux actions in this file can probably be removed as part of this change? But maybe you prefer to leave that for a separate PR?

Comment (zhuje) [reply frontend/public/reducers/observe.ts]: Thnx! I removed the `export` in the latest commit and create a separate issue to remove the redux actions [OU-555](https://issues.redhat.com/browse/OU-555).

**Comment by zhuje** [comment]
Please review:
QE:
/assign @juzhao
Docs:
/assign @opayne1
PX:
/assign @reestr

**Comment by zhuje** [comment]
/px-approved

/docs-approved 

This PR does not change experience or documentation.

**Comment by zhuje** [comment]
/label docs-approved
  
/label px-approved

This PR does not change experience or documentation.

**Comment by zhuje** [comment]
Please review, 
/assign @yapei
